docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: '2.1'
+
+services:
+  postgres:
+    user: postgres
+    image: postgres
+    ports: 
+      - "5432:5432"
+    environment:
+      POSTGRES_USER: 'postgres'
+      POSTGRES_DB: 'elasticapm_test'
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready"]
+      interval: 30s
+      timeout: 30s
+      retries: 3
+
+  mongodb:
+    image: mongo
```

Comment:
add a specific version? Same applies for all other images.

History is that in metricbeat we didn't have this initially but over time new and old builds started to fail because there was a new `mongodb` version.